doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -3291,26 +3291,31 @@ The following table indicates the software maturity levels of the support for ea
               - nRF54L05
               - nRF54L10
               - nRF54L15
+              - nRF54LM20
             * - **nPM1100**
               - --
               - --
               - --
               - --
+              - --
             * - **nPM1300**
               - Supported
               - --
               - --
               - Supported
+              - Supported
```

Comment:
Is this entry correct? Could find the support here: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/pmic/npm1300.html#boards_compatible_with_npm1300

Reply:
We have board files for the nRF54LM20 in https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/pmic/native/npm13xx_fuel_gauge/README.html and https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/pmic/native/npm13xx_one_button/README.html

I can update that table as well to reflect that

Reply:
> I think we could also add nPM1304 to this table.

Yes, good point. Adding it now